src/applications/personalization/rated-disabilities/util/index.js

```diff
@@ -8,3 +11,7 @@ export async function getData(apiRoute, options) {
     return error;
   }
 }
+
+export const isServerError = errCode => SERVER_ERROR_REGEX.test(errCode);
```

Comment:
I decided to just move these into utilities because they were getting reused quite a bit.

Reply:
These new helpers are begging for unit tests :)